doc/nrf/dev_model_and_contributions/documentation/styleguide.rst

```diff
@@ -142,6 +142,172 @@ To link to the Kconfig options from RST, use the ``:kconfig:option:`` domain::
 
    :kconfig:option:`CONFIG_DEBUG`
 
+Kconfig |gl|
+************
+
+The |NCS| documentation follows Zephyr's :ref:`zephyr:kconfig_style` and :ref:`zephyr:kconfig_tips_and_tricks`, with the addition of the following rules.
+
+See Zephyr's :ref:`zephyr:kconfig` for a short introduction to Kconfig.
+
+Additions to file structure
+===========================
+
+In addition to the Zephyr guidelines for :ref:`File Organization <zephyr:kconfig_style>`, observe the following guidelines in the |NCS|:
+
+* Always include the Nordic Semiconductor copyright header at the top of the Kconfig file in the |NCS|.
+
+  .. code-block:: kconfig
+
+     #
+     # Copyright (c) YEAR Nordic Semiconductor
+     #
+     # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+     #
+
+* For subsystems and libraries, include the standard logging template:
+
+   .. code-block:: kconfig
+
+      module = MODULE_NAME
+      module-str = Module description
+      source "$(ZEPHYR_BASE)/subsys/logging/Kconfig.template.log_config"
+
+  Place this at the end of the module's configuration section, before the final ``endif``.
+
+Additions to symbol naming and structure
+========================================
+
+In addition to the Zephyr guidelines for :ref:`Symbol Naming and Structure <zephyr:kconfig_style>`, observe the following guidelines in the |NCS|:
+
+* All configuration options with prompts must have the help text.
+* Keep prompts concise - detailed information belongs in the help text.
+* Avoid redundant phrases like "Enable support for" - just use the feature name.
```

Comment:
Not avoid, do not use at all, symbols starting with "Enable" are completely invalid and will fail CI, unfortunately those checks do not run on test/sample Kconfig files though